armi/utils/directoryChangersMpi.py

```diff
@@ -54,6 +54,7 @@ def open(self):
 
     def close(self):
         cdma = _ChangeDirectoryMpiAction(self.initial)
+        cdma = cdma.broadcast(cdma)
```

Comment:
If you want to prove to yourself that this was bugged before, remove this line and run the new unit test that I put up with this PR.